src/renderer/src/components/AddItemForm.jsx

```diff
@@ -12,9 +12,9 @@ function AddItemForm() {
     const newItem = {
       text: inputRef.current.value,
       key: Date.now(),
-      status: "pending"
+      status: "pending",
     };
-    if (!!newItem.text.trim()) {
+    if (newItem.text.trim()) {
```

Comment:
If I recall, the `!!` was important, but I don't remember enough to axe this, so I'll just call this good and add it back if needed after doing some more testing